guide/14-deep-learning/how_AutoDL_works.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<p>This guide has explained how the <code>AutoDL</code> class can be used to automate multiple deep learning models supported by the<code>arcgis.learn</code> API. For every step in the workflow, we defined a function and discussed its usage. This guide can be a starting point for developers to train and evaluate multiple <code>arcgis.learn</code> supported models' performances.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>